packages/sdk/odahuflow/sdk/clients/api.py

```diff
@@ -384,6 +386,16 @@ def __init__(self,
         self.authenticator = Authenticator(client_id, client_secret, non_interactive, base_url, token)
         self.conn_policy = HTTPConnectionPolicy(retries, timeout)
 
+        retry_strategy = Retry(
+            total=self.conn_policy.get_left_retries(),
+            status_forcelist=[429, 500, 502, 503, 504],
+            backoff_factor=1
+        )
+        adapter = HTTPAdapter(max_retries=retry_strategy)
+        self.default_client = requests.Session()
+        self.default_client.mount("https://", adapter)
+        self.default_client.mount("http://", adapter)
```

Comment:
Not sure, but I think it would work just the same. If for some reason we try to reach non-http(s) URI, well whatever :) 
```suggestion
        self.default_client.mount(prefix='', adapter=adapter)
```